moq-sub/src/main.rs

```diff
@@ -0,0 +1,92 @@
+use std::{net, time};
+
+use anyhow::Context;
+use clap::Parser;
+use url::Url;
+
+use moq_native::quic;
+use moq_sub::media::Media;
+use moq_transport::serve::Tracks;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+	env_logger::init();
+
+	// Disable tracing so we don't get a bunch of Quinn spam.
+	let tracer = tracing_subscriber::FmtSubscriber::builder()
+		.with_max_level(tracing::Level::WARN)
+		.finish();
+	tracing::subscriber::set_global_default(tracer).unwrap();
+
+	let config = Config::parse();
+
+	let out = tokio::io::stdout();
+
+	let tls = config.tls.load()?;
+
+	let quic = quic::Endpoint::new(quic::Config { bind: config.bind, tls })?;
+
+	let session = quic.client.connect(&config.url).await?;
+
+	let (session, subscriber) = moq_transport::session::Subscriber::connect(session)
+		.await
+		.context("failed to create MoQ Transport session")?;
+
+	// Associate empty set of Tracks with provided namespace
+	let tracks = Tracks::new(config.name);
+
+	let mut media = Media::new(subscriber, tracks, out).await?;
+
+	tokio::select! {
+		res = session.run() => res.context("session error")?,
+		res = media.run() => res.context("media error")?,
+	}
+
+	Ok(())
+}
+
+pub struct NoCertificateVerification {}
```

Comment:
I don't think this is needed any longer.

Reply:
Ack. Let's clean that up post-merge.